src/rust/plugin-registry/src/server.rs

```diff
@@ -124,7 +129,12 @@ impl PluginRegistryService for PluginRegistry {
         &self,
         _request: Request<GetGeneratorsForEventSourceRequestProto>,
     ) -> Result<Response<GetGeneratorsForEventSourceResponseProto>, Status> {
-        todo!()
+        // Stub implementation, for integration test smoke-test purposes.
+        // Replace with a real implementation soon!
+        let message = _request.into_inner();
+        Ok(Response::new(GetGeneratorsForEventSourceResponseProto {
+            plugin_ids: [message.event_source_id].into_iter().flatten().collect(),
+        }))
```

Comment:
It might make sense to just leave this as `todo` and mark the test as `#[should_panic]` or something

Reply:
_yes_ but I want to test that we get a real response. should_panic could result from any number of things, like the client not working. should_panic doesn't guarantee we have a real connection

Reply:
i think a response that's explicitly "yo this is a fake" would be the better solution